src/main/java/uk/gov/companieshouse/web/pps/models/AvailablePenaltyReference.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.companieshouse.web.pps.models;
+
+import java.util.ArrayList;
+import java.util.List;
+import uk.gov.companieshouse.web.pps.util.PenaltyReference;
+
+public class AvailablePenaltyReference {
+    private List<PenaltyReference> availablePenaltyReference;
+
+    public List<PenaltyReference> getAvailablePenaltyReference() {
+        availablePenaltyReference = new ArrayList<>();
+        availablePenaltyReference.add(PenaltyReference.LATE_FILING);
+        availablePenaltyReference.add(PenaltyReference.SANCTIONS);
+        return availablePenaltyReference;
```

Comment:
Although I know this code is hit in one of the other tests, I would expect to see a unit test for this.